src/plugins/dmt-app/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@dmt/dmt-app",
-  "version": "0.0.1",
+  "version": "0.0.2",
```

Comment:
I think it would be easier to skip the "DMT" plugin at this point.
And simply have a "dmtApplication.json" that renders the "header" plugin.